summary_workflows/quantification/quantification_dockers/q_consolidation/manage_assessment_data.py

```diff
@@ -80,7 +166,7 @@ def generate_manifest(data_dir,output_dir, participant_data):
                 metric_Y = "precision"
             
             aggregation_file = {
-                "_id": "TCGA:{}_{}_Aggregation".format(DEFAULT_eventMark, challenge),
+                "_id": "{}:{}_{}_Aggregation".format(metrics_file[0]["community_id"],event_date, challenge),
                 "challenge_ids": [
                     challenge
                 ],
```

Comment:
after line 175 (after challenge participants), the TCGA script contains the following:
```python
"metrics":{
    "metric_x_id": METRICS['TPR'],
    "metric_y_id": METRICS['precision']
},
```
That's one place where METRICS is used. Is it needed? Can't find key `metric_x_id` anywhere

Reply:
We don't need `metrics` or any of its children (`metric_x_id`, `metric_y_id`), this was related to the test that a colleague did that I mentioned [above](https://github.com/iRNA-COSI/APAeval/pull/182#discussion_r852898176).